pkg/events/events.go

```diff
@@ -0,0 +1,120 @@
+package events
+
+import (
+	k8sV1 "k8s.io/api/core/v1"
+
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+)
+
+const AnyNetworkPlugin = "any"
+
+type EventHandler interface {
```

Comment:
```suggestion
type Handler interface {
```
I also suggest to rename the package to `event` so references read `event.Handler` which I think is cleaner and avoids stutter.